src/templates/toolkit/global/head.html.tt

```diff
@@ -5,6 +5,20 @@
 <html>
 [% END %]
     <head>
```

Comment:
I don't think you need to add to the toolkit version. We only redirect to www2 when the client hits the homepage.